src/mqueryfront/src/QueryResultsStatus.js

```diff
@@ -226,6 +230,13 @@ class QueryResultsStatus extends Component {
                     >
                         {progress}%
                     </div>
+                    <div
+                        className={"progress-bar warning"}
+                        role="progressbar"
+                        style={{ width: processing + "%" }}
+                    >
+                        {processing}%
```

Comment:
what if processing% < 1? (it's rounded down to integer, so even 0.99% will show as 0%).

If processing is equal to 0% we probably shouldn't show this progress bar at all?

Also maybe consider rounding the processing to 2 decimal digits instead of to integers?

Also if the processing % is very small, it'll be too thin (narrower than the text) and it looks ugly. Maybe consider setting a minimum value for display width? (for example minimum 3%, so if `processsing > 0 && processing < 3`, use 3% for width).